index.js

```diff
@@ -32,4 +32,21 @@ module.exports = options => options ?
 	getPort(options).catch(() => getPort(Object.assign(options, {port: 0}))) :
 	getPort({port: 0});
 
+module.exports.makeRange = (from, to) => {
+	if (!Number.isInteger(from) || !Number.isInteger(to) || from <= 1024 || to <= 1024 || from > 65535 || to > 65535) {
```

Comment:
This condition is omitting a range-like error, i.e., `RangeError`, no?

Reply:
done